common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -526,18 +526,6 @@ def test_download_transcript_button_works_correctly(self):
         # check if we can download transcript in "srt" format that has text "00:00:00,260"
         self.assertTrue(self.video.downloaded_transcript_contains_text('srt', '00:00:00,260'))
 
-        # select the transcript format "txt"
```

Comment:
I've removed it intentionally. Do we need to test for both? If so, it'll cause a re-write of the entire test because the whole method changed.